docs/en/observability/analyze-monitors.asciidoc

```diff
@@ -1,6 +1,7 @@
 [[analyze-monitors]]
 = Analyze monitors
 
+// lint ignore observability
 To access this page, go to *Observability > Uptime > Monitors*. Click on a listed monitor
```

Comment:
There are many inline overrides for `Observability` in this context because it's referencing the UI. Do you think it's safe to use the `{observability}` attribute in these cases since this particular piece of UI is so tied to the name of the solution?

Reply:
Ok I removed these overrides in https://github.com/elastic/observability-docs/pull/1820/commits/55f5c72914496c77e1b7c2cfd60165388d451d04. I also removed overrides for `{fleet}` where we're referencing the Fleet item in Kibana's main menu (https://github.com/elastic/observability-docs/pull/1820/commits/03fdea34bd3274757bfb574a55b2af4bc05bcbb5). I did these separately so we can easily revert if we want to.